static/app/gettingStartedDocs/go/gin.tsx

```diff
@@ -31,14 +31,26 @@ import (
 
 // To initialize Sentry's handler, you need to initialize Sentry itself beforehand
 if err := sentry.Init(sentry.ClientOptions{
-  Dsn: "${params.dsn}",
+  Dsn: "${params.dsn}",${
+    params.isPerformanceSelected
+      ? `
   EnableTracing: true,
   // Set TracesSampleRate to 1.0 to capture 100%
   // of transactions for performance monitoring.
   // We recommend adjusting this value in production,
-  TracesSampleRate: 1.0,
+  TracesSampleRate: 1.0,`
```

Comment:
A few of these go platforms have `EnableTracing: true` as well as `TracesSampleRate: 1.0`. Do we need both? @cleptric do you know?

Reply:
Yes, to enable performance with an empty sample rate to only sample on incoming requests. Go‘s null values required this.